master/internal/api_experiment.go

```diff
@@ -92,6 +92,33 @@ func (a *apiServer) GetExperiments(
 		if len(req.Users) != 0 && !found {
 			return false
 		}
+
+		if len(req.Labels) != 0 {
+			found = true
+
+			if len(v.Labels) == 0 {
+				return false
+			}
+
+			for _, reqLabel := range req.Labels {
+				hasAMatch := false
+				for _, expLabel := range v.Labels {
+					if hasLabel := strings.Contains(strings.ToLower(expLabel), strings.ToLower(reqLabel)); hasLabel {
+						hasAMatch = true
+						break
+					}
+				}
+				if !hasAMatch {
+					found = false
+					break
+				}
+			}
```

Comment:
question: Is this a logical AND of the labels in the request, or a logical OR of the labels?

Reply:
AND. https://determined-ai.slack.com/archives/C013B86TBPC/p1599604207019900